extensions/banner/runtime/pom.xml

```diff
@@ -0,0 +1,48 @@
+<project xmlns='http://maven.apache.org/POM/4.0.0'
+         xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+         xsi:schemaLocation='http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd'>
+
+
+    <parent>
+        <artifactId>quarkus-banner-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-banner</artifactId>
+    <name>Quarkus - Banner - Runtime</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core-deployment</artifactId>
```

Comment:
runtime module should not depend on deployment module.

Reply:
Very very good catch

Reply:
Why so ? 😅

Reply:
The runtime module should never depend on anything from the deployment module. The deployment module is meant for build time only code and dependencies